accounts/abi/bind/backends/simulated.go

```diff
@@ -109,20 +122,21 @@ func NewXDCSimulatedBackend(alloc core.GenesisAlloc, gasLimit uint64, chainConfi
 	genesis.MustCommit(database)
 	consensus := XDPoS.NewFaker(database, chainConfig)
 
-	// Attach mock trading and lending service
-	var DefaultConfig = XDCx.Config{
-		DataDir: "",
-	}
-	XDCXServ := XDCx.New(&DefaultConfig)
-	lendingServ := XDCxlending.New(XDCXServ)
-
-	consensus.GetXDCXService = func() utils.TradingService {
-		return XDCXServ
-	}
-	consensus.GetLendingService = func() utils.LendingService {
-		return lendingServ
-	}
+	/*
+		// Attach mock trading and lending service
```

Comment:
why not just delete them?

Reply:
hmm, initially thought we will gonna use it in the future, but you are right. I can add it back whenever we need it